android/src/main/java/com/swmansion/rnscreens/RNScreensPackage.kt

```diff
@@ -4,10 +4,19 @@ import com.facebook.react.ReactPackage
 import com.facebook.react.bridge.NativeModule
 import com.facebook.react.bridge.ReactApplicationContext
 import com.facebook.react.uimanager.ViewManager
+import com.facebook.soloader.SoLoader
 
 class RNScreensPackage : ReactPackage {
-    override fun createNativeModules(reactContext: ReactApplicationContext) =
-        emptyList<NativeModule>()
+    override fun createNativeModules(reactContext: ReactApplicationContext): List<NativeModule> {
+        if (BuildConfig.IS_NEW_ARCHITECTURE_ENABLED) {
+            // For Fabric, we load c++ naitve library here, this triggers screen's Fabric
```

Comment:
native

Reply:
Fixed in [bc3f058](https://github.com/software-mansion/react-native-screens/pull/1322/commits/bc3f0588d91028fd98f5b9eeff12134678002d13)